src/actions/index.js

```diff
@@ -8,4 +8,9 @@ const REMOVE_BOOK = (book) => ({
   text: book.id,
 });
 
-export { CREATE_BOOK, REMOVE_BOOK };
+const CHANGE_FILTER = (category) => ({
```

Comment:
### [Optional]

Best practice in react action, our method name should not be in all caps.
It would be nice like this 👇 
```
const changeFilter = (category) => ({
    type: 'CHNAGE_FILTER',
.....
```